README.md

```diff
@@ -24,13 +24,13 @@ If you have found a bug elsewhere in the snap world:
 
 ## Local development
 
-The simplest way to run the site locally is to first [install Docker](https://docs.docker.com/engine/installation/) (on Linux you may need to [add your user to the `docker` group](https://docs.docker.com/engine/installation/linux/linux-postinstall/)), and then use the `./run` script:
+The simplest way to run the site locally is using the [`dotrun`](https://github.com/canonical-web-and-design/dotrun/) and [`docker`](https://snapcraft.io/docker)snaps:
```

Comment:
Probably best to just link to the install link https://snapcraft.io/dotrun which provides a link to the github project if someone needs it. They don't need docker.